documentation/developing-your-hyrax-based-app.md

```diff
@@ -269,3 +270,158 @@ proxy_deposit:
 
 If both options exist, whichever option is set from the Administrative Dashboard
 will take precedence.
+
+## Dockerizing a Hyrax Based App for Development
+It is now possible develop a Hyrax based app using a Docker workflow. The process utilizes **docker-compose** to build and run the services necessary for Hyrax based applications.
+
+### Switch to Postgres
+The image that you will build your app's image on uses **Postgres**. If you have just created your Hyrax-based app, for example, you'll need to update your `Gemfile`
+	- `gem "sqlite3", "~> 1.3.0"` ---> `gem "pg"`
+
+### Building the image
+Next you'll want to build the image that your Hyrax container will run from. For that, you will need to create a Dockerfile, build the image from it, and push it to a repository you can fetch from later.
+
+**Build image**
+- Run `cd <hyrax_app_name>`
+- Visit [samveralabs/hyrax](https://hub.docker.com/r/samveralabs/hyrax/tags) and grab the latest image **tag**
+- Run `echo "FROM samveralabs/hyrax:7<tag>" > Dockerfile` to create the Dockerfile
+- Run `docker build .`
+
+**Push image to a repo (assumes dockerhub.io)**
```

Comment:
the push to a web-available registry is needed for the kubernetes/helm deployments, but i think it can be streamlined out here.

it's probably best in most cases to tag locally on build `docker build . -t my_app:dev`, and then use `my_app:dev` in place of `<sha-you-copied>` and `<your-image-location>`.